app/src/main/java/org/p2p/wallet/user/interactor/TokenMetadataInteractor.kt

```diff
@@ -62,6 +82,7 @@ class TokenMetadataInteractor(
 
         if (tokensMetadata?.tokens == null) {
             Timber.tag(TAG).e("Local file not found!")
+            subscribeOnInternetConnection()
```

Comment:
can we do it in the splash screen? Jobs and stuff is not domain stuff

Reply:
splash will die
metadata needed on Main Screen with reload